src/ema_cross_strategy.py

```diff
@@ -0,0 +1,75 @@
+import indicator_lib
+import mt5_lib
+
+def ema_cross_strategy(symbol, timeframe, ema_one, ema_two):
```

Comment:
Let's rename our ema_one and ema_two to reference the length of each. Something like short_term_ema and long_term_ema